### PR TITLE
Change javadocs links to javadoc.io

### PR DESCRIPTION
**Pull request by Frosty-J**
As discussed on Discord, the javadocs hosted at libgdx.badlogicgames.com are outdated. While generating our own javadocs and hosting them at libgdx.com would be nice, this is an adequate stopgap in the meantime.

Not all links are tested but I've been trying random ones and it seems okay. If you go to a page that doesn't exist, javadoc.io soft-404s, so pay attention when testing.

Related: https://github.com/libgdx/libgdx/pull/6876

### CODE REVIEW

File: wiki/graphics/2d/tile-maps.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Tile maps
 ---
 # Maps
 
-libGDX features a generic maps API. All map related classes can be found in the [com.badlogic.gdx.maps](https://libgdx.badlogicgames.com/nightlies/docs/api/index.html?com/badlogic/gdx/maps/package-use.html) [(code)](https://github.com/libgdx/libgdx/tree/master/gdx/src/com/badlogic/gdx/maps) package. The root package contains the base classes, sub-packages contain specialized implementations for tile maps and other forms of maps.
+libGDX features a generic maps API. All map related classes can be found in the [com.badlogic.gdx.maps](https://libgdx.badlogicgames.com/ci/nightlies/docs/api/com/badlogic/gdx/maps/package-use.html) [(code)](https://github.com/libgdx/libgdx/tree/master/gdx/src/com/badlogic/gdx/maps) package. The root package contains the base classes, sub-packages contain specialized implementations for tile maps and other forms of maps.
```
Comment (Frosty-J) [inline wiki/graphics/2d/tile-maps.md:6]: This one is intentionally left at libgdx.badlogicgames.com due to no package use pages at javadocs.io or in our Maven JARs. Lyze is investigating.

Comment (Frosty-J) [reply wiki/graphics/2d/tile-maps.md:6]: Probably solved in https://github.com/libgdx/libgdx/pull/6877. But there's no point changing that link until 1.11.1 is out, as we'd just have a broken link until then.

**Comment by crykn** [comment]
Thanks!

File: wiki/graphics/2d/tile-maps.md (hunk 1 above)
Comment (crykn) [reply wiki/graphics/2d/tile-maps.md:6]: Okay, so we'll have to keep that in mind when 1.11.1 releases.

**Pull request merged**